setupext.py

```diff
@@ -2165,3 +2165,22 @@ def check(self):
             pass
 
         raise CheckFailed()
+
+
+class OptionalPackageData(OptionalPackage):
+    config_category = "package_data"
+
+
+class Dlls(OptionalPackageData):
+    """
+    On Windows, this packages any DLL files that can be found in the 
+    lib/matplotlib/* directories.
+    """
+    name = "dlls"
+
+    def check_requirements(self):
+        if sys.platform != 'win32':
```

Comment:
Incidentally, is it preferable to use ```platform.system```? I confess, I've tried in the past to understand what the win32 means, but it always escapes me - as far as I can remember, even 64-bit windows reports win32, right?